src/window.rs

```diff
@@ -42,7 +42,26 @@ pub struct Attributes {
     pub type_hint: Option<WindowTypeHint>,
 }
 
-impl Attributes {
+impl Default for WindowAttr {
+    fn default () -> WindowAttr {
+        WindowAttr {
+            title: None,
+            event_mask: 0,
+            x: None,
+            y: None,
+            width: 800,
+            height: 600,
```

Comment:
These a are bit concerning to me. Wouldn't 'neutral' values be smaller somewhere between 100 and 400?